incubating/wrappers/s2i/java-jni/java-jni/JavaJNIServer.py

```diff
@@ -0,0 +1,50 @@
+import os
+import logging
+import jpype
+
+from seldon_core.user_model import SeldonComponent
+
+logger = logging.getLogger(__name__)
+
+
+class JavaJNIServer(SeldonComponent):
+    def __init__(self):
+        super().__init__()
+        self._model = None
+
+    def load(self):
+        """
+        We can only have a single JVM per process.
+        More details can be found here:
+        https://jpype.readthedocs.io/en/latest/userguide.html#multiprocessing
+        """
+        model_jar_path = os.getenv("JAVA_JAR_PATH")
+
+        logger.debug(f"Starting JVM with jar: {model_jar_path}")
+        # NOTE: convertStrings must be set to True to avoid an explosion of
+        # interop calls when working with the returned Java strings
+        jpype.startJVM(classpath=[model_jar_path], convertStrings=True)
+
+        # TODO: Make class name configurable
```

Comment:
Better to move to github issue than todo in code

Reply:
Good spot! That one isn't actually needed anymore.